pydicom/tests/test_charset.py

```diff
@@ -11,16 +11,16 @@
 import os.path
 import pydicom.charset
 
-test_dir = os.path.dirname(__file__)
-testcharset_dir = os.path.join(test_dir, 'charset_files')
+test_dir, _ = os.path.split(os.path.dirname(__file__))
```

Comment:
I would suggest also that:

`test_dir, _ = os.path.split(os.path.dirname(__file__))
` 
be

`test_dir = os.path.dirname(os.path.dirname(__file__)) 
`

in general.